with-snowflake/stepzen.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "endpoint": "api/snowemp"
```

Comment:
Should be `with-snowflake` or `with-snowemp` based on how other endpoints are named

Reply:
fixed